app/models/indeed_api.rb

```diff
@@ -17,10 +17,22 @@ def self.search_jobs(search_params)
           fromage: "any",
           highlight: 1,
           latlong: 1,
-          # userip: user_ip,
-          # useragent: user_browser,
           v: 2
-          }
+        }
+      }
+    )
+  end
+
+  def self.search_for_job(search_param)
```

Comment:
Is this really a "search"  search generally implies many varied results maybe something like `find_job`?.